exercises/outcomes/AT/AT5/generator.sage

```diff
@@ -61,26 +82,19 @@ class Generator(BaseGenerator):
             a=randrange(1,8)
             theta = lambda v : vector([v[0],v[1]+a])
             untheta = lambda v : vector([v[0],v[1]-a])
-
-            trueproperty=choice(["dist_v","dist_s"])
-            falseproperties=[
-                "mul_assoc",
-                "mul_id",
-            ]
+            true_no_check_properties.append("add_comm") #Too easy
 
         elif n==2:            
             plus = lambda v1,v2 : vector([v1[0]+v2[0], v1[1]+v2[1]])
             r2 = randrange(2,4)
```

Comment:
![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/b7c3e876-3064-4112-b48c-f2b7660db3e5)

This seems like a bit much.

Reply:
addressed in 5160683